ocs_ci/ocs/scale_lib.py

```diff
@@ -998,3 +979,249 @@ def get_max_pvc_count():
+                    del pod_data["spec"]["template"]["spec"]["containers"][0]["stdin"]
+                    del pod_data["spec"]["template"]["spec"]["containers"][0]["tty"]
+                    flag = 0
+                elif flag:
+                    # Update pod yaml with liveness probe and IO
+                    pod_data["spec"]["containers"][0]["args"] = [
+                        "/bin/sh",
+                        "-c",
+                        f"fio --name=fio-rand-readwrite --filename={liveness_check_path}/abc "
+                        f"--readwrite=randrw --bs=4K --direct=1 --numjobs=1 --time_based=1 "
+                        f"--runtime=3600000 --size=512M --iodepth=4 --fsync_on_close=1 "
+                        f"--rwmixread=25 --ioengine=libaio --rate=2k",
+                    ]
+                    liveness = {
+                        "exec": {"command": ["sh", "-ec", "df /mnt"]},
+                        "initialDelaySeconds": 3,
+                        "timeoutSeconds": 10,
+                    }
+                    pod_data["spec"]["containers"][0]["livenessProbe"] = liveness
+                    del pod_data["spec"]["containers"][0]["command"]
+                    del pod_data["spec"]["containers"][0]["stdin"]
+                    del pod_data["spec"]["containers"][0]["tty"]
+                    flag = 0
+
+                if node_selector:
+                    pod_data["spec"]["template"]["metadata"]["labels"] = node_selector
+
+            temp_list.clear()
+            pods_list.append(pod_data)
+
+    return pods_list
+
+
+def get_pod_creation_time_in_kube_job(kube_job_obj, namespace, no_of_pod):
+    """
+    Function to get pod creation time of pods created using kube_job
+    Note: Function doesn't support DeploymentConig pods
+
+    Args:
+        kube_job_obj (obj): Kube Job Object
+        namespace (str): Namespace of PVC's created
+        no_of_pod (int): POD count
+
+    Return:
+        pod_dict (dict): Dictionary of pod_name with creation time.
+
+    """
+    job_get_output = kube_job_obj.get(namespace=namespace)
+    pod_dict = dict()
+    for i in range(no_of_pod):
+        started_at_str = job_get_output["items"][i]["status"]["containerStatuses"][0][
+            "state"
+        ]["running"]["startedAt"]
+        start_time_str = job_get_output["items"][i]["status"]["startTime"]
+        started_at = re.search(r"(\d\d):(\d\d):(\d\d)", started_at_str)
+        started_at = started_at[0]
+        start_time = re.search(r"(\d\d):(\d\d):(\d\d)", start_time_str)
+        start_time = start_time[0]
+        format = "%H:%M:%S"
+        pod_start_at = datetime.datetime.strptime(started_at, format)
+        pod_start_time = datetime.datetime.strptime(start_time, format)
+        total = pod_start_at - pod_start_time
+        pod_name = job_get_output["items"][i]["metadata"]["name"]
+        pod_dict[pod_name] = total.total_seconds()
+
+    return pod_dict
```

Comment:
These 3 functions are part of PR #4104 , without this function can't run validation job, will be rebase once the PR is merged.